PendingReleaseNotes

```diff
@@ -13,59 +13,65 @@
+  Existing clusters must still set the "enable_multiple" flag on the fs. 
+  See the CephFS documentation for more information.
+
+* volume/nfs: The "ganesha-" prefix from cluster id and nfs-ganesha common
+  config object was removed to ensure a consistent namespace across different
+  orchestrator backends. Delete any existing nfs-ganesha clusters prior
   to upgrading and redeploy new clusters after upgrading to Pacific.
 
-* A new health check, DAEMON_OLD_VERSION, will warn if different versions of Ceph are running
-  on daemons. It will generate a health error if multiple versions are detected.
-  This condition must exist for over mon_warn_older_version_delay (set to 1 week by default) in order for the
-  health condition to be triggered.  This allows most upgrades to proceed
-  without falsely seeing the warning.  If upgrade is paused for an extended
-  time period, health mute can be used like this
-  "ceph health mute DAEMON_OLD_VERSION --sticky".  In this case after
-  upgrade has finished use "ceph health unmute DAEMON_OLD_VERSION".
-
-* MGR: progress module can now be turned on/off, using the commands:
+* A new health check, DAEMON_OLD_VERSION, warns if different versions of
+  Ceph are running on daemons. It generates a health error if multiple
+  versions are detected.  This condition must exist for over 
+  ``mon_warn_older_version_delay`` (set to 1 week by default) in order for the 
+  health condition to be triggered. This allows most upgrades to proceed 
+  without falsely seeing the warning.  If upgrade is paused for an extended 
+  time period, health mute can be used like this "ceph health mute 
+  DAEMON_OLD_VERSION --sticky". In this case after upgrade has finished use 
+  "ceph health unmute DAEMON_OLD_VERSION".
+
+* MGR: progress module can now be turned on/off, using these commands:
   ``ceph progress on`` and ``ceph progress off``.
 
 * The ceph_volume_client.py library used for manipulating legacy "volumes" in
   CephFS is removed. All remaining users should use the "fs volume" interface
   exposed by the ceph-mgr:
   https://docs.ceph.com/en/latest/cephfs/fs-volumes/
 
-* An AWS-compliant API: "GetTopicAttributes" was added to replace the existing "GetTopic" API. The new API
-  should be used to fetch information about topics used for bucket notifications.
+* An AWS-compliant API: "GetTopicAttributes" was added to replace the existing 
+  "GetTopic" API. The new API should be used to fetch information about topics 
+  used for bucket notifications.
 
-* librbd: The shared, read-only parent cache's config option ``immutable_object_cache_watermark`` now has been updated
-  to property reflect the upper cache utilization before space is reclaimed. The default ``immutable_object_cache_watermark``
-  now is ``0.9``. If the capacity reaches 90% the daemon will delete cold cache.
+* librbd: The shared, read-only parent cache's config option 
+  ``immutable_object_cache_watermark`` has now been updated to property reflect 
```

Comment:
s/property/properly/

Reply:
I've made this change -- s/property/properly/

Good catch, Josh.